netplugin/external_networker.go

```diff
@@ -265,6 +265,11 @@ func (p *externalBinaryNetworker) exec(log lager.Logger, action, handle string,
 		}
 	}
 
-	log.Debug("external-networker-result", logData)
+	if stderr.String() == "" {
+		log.Debug("external-networker-result", logData)
+	} else {
+		log.Info("external-networker-result", logData)
+	}
```

Comment:
Not sure I like changing the log level for all of logData based on a buffer's content.

How would you feel about instead adding something like this after line 258:

```
if stderr.Len() > 0 {
    log.Info("external-networker-result", lager.Data{"stderr": stderr.String()})
}
```

Reply:
Sounds reasonable to me. I've updated the PR with the change.